src/main/java/com/example/todolist/API/Restful/RestController/AuthController.java

```diff
@@ -35,13 +39,28 @@ public ResponseEntity<?> signUp(@RequestBody @Valid SignUpRequest signUpRequest,
             HttpSession session = request.getSession();
             session.setAttribute("user_id", user.getId());
 
-            return ResponseEntity.status(201).body(new SignUpResponse());
+            return ResponseEntity.status(201).body(new Response("Sign up successful"));
+
         } catch (DuplicatedUsernameException e) {
             ApiError error = ApiError.FromHasErrorCodeExceptionToApiError("username", e);
             return ResponseEntity.badRequest().body(error);
+
         } catch (DuplicatedUserEmailException e) {
             ApiError error = ApiError.FromHasErrorCodeExceptionToApiError("email", e);
             return ResponseEntity.badRequest().body(error);
         }
     }
+
+    @PostMapping("/sign-in")
+    public ResponseEntity<?> signIn(@RequestBody @Valid SignInRequest signInRequest, HttpServletRequest request) {
+        User user = userRepository.findUserByEmailAndPassword(signInRequest.email(), signInRequest.password());
+        if (user == null) {
+            return ResponseEntity.badRequest().body(new Response("Sign in failed"));
+        } else {
```

Comment:
using else here is unnecessary, since the previous code block use `return` statement, find a better way to write this code